galaxy/tools/deps/conda_util.py

```diff
@@ -220,8 +220,12 @@ def exec_command(self, operation, args):
         env = {}
         if self.condarc_override:
             env["CONDARC"] = self.condarc_override
-        cmd_string = ' '.join(map(shlex_quote, cmd))
-        log.debug("Executing command: %s", cmd_string)
+        if ">" not in cmd:
+            cmd_string = ' '.join(map(shlex_quote, cmd))
+            log.debug("Executing command: %s", cmd_string)
+        else:
+            cmd_string = ' '.join(cmd)
+            cmd = cmd_string
```

Comment:
I think this may introduce a security vulnerability if an attacker is able to include `>` among `args`. I'd rather break galaxy-lib API, if this is the case.

Reply:
:( but it wasn't broken and it didn't have known bugs before. 

I'll see if I can take another stab at this but it means Planemo is broken for longer.